### PR TITLE
Update horizontal-pod-autoscale-walkthrough.md

### PR DESCRIPTION
**Pull request by jiaj12**
fix typo and broken links

### CODE REVIEW

**Comment by tengqm** [comment]
What is this trying to fix?

File: content/zh/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ For more information on how Horizontal Pod Autoscaler behaves, see the
 -->
 本文将引领你了解如何为 php-apache 服务器配置和使用 Horizontal Pod Autoscaler。
 与 Horizontal Pod Autoscaler 相关的更多信息请参阅
-[Horizontal Pod Autoscaler 用户指南](/zh/docs/tasks/run-application/horizontal-pod-autoscale/)。
+[Horizontal Pod Autoscaler 用户指南](/docs/tasks/run-application/horizontal-pod-autoscale/)。
```
Comment (helight) [inline content/zh/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md:36]: https://kubernetes.io/zh/docs/tasks/run-application/horizontal-pod-autoscale/

这个地址是存在的呀

File: content/zh/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md [hunk 2]
```diff
@@ -149,7 +149,7 @@ Deployment，使 Pod 的副本数量维持在 1 到 10 之间。
 大致来说，HPA 将（通过 Deployment）增加或者减少 Pod 副本的数量以保持所有 Pod
 的平均 CPU 利用率在 50% 左右（由于每个 Pod 请求 200 毫核的 CPU，这意味着平均
 CPU 用量为 100 毫核）。
-算法的详情请参阅[相关文档](/zh/docs/tasks/run-application/horizontal-pod-autoscale/#algorithm-details)。
```
Comment (helight) [inline content/zh/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md:152]: 这些地址都是存在的

Comment (jiaj12) [reply content/zh/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md:152]: 原文链接无法正常访问，您可以试一下，可能是同层目录下无法正常识别，您不需要在路径前添加zh

Comment (helight) [reply content/zh/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md:152]: https://kubernetes.io/zh/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/
这里看一下，都是ok的。

**Comment by tengqm** [comment]
And ...  this is targeting release-1.19, why is that?
The "master" branch we see today is for 1.19, until we release 1.20.

File: content/zh/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md (hunk 2 above)
Comment (jiaj12) [reply content/zh/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md:152]: 文档本身是在content/zh/或content/en/路径下，所以在文章内的/docs/tasks/run-application/xxx格式的相对路径是可以被访问到，原文/zh/docs/tasks/run-application/horizontal-pod-autoscale样式的相对路径被会被映射为类似https://kubernetes.io/zh/zh/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/，所以会显示为404

File: content/zh/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md (hunk 1 above)
Comment (jiaj12) [reply content/zh/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md:36]: 抱歉 没有跟上最新的翻译要求

**Comment by jiaj12** [comment]
It's my fault for not following the new rules, closing this pr

**Pull request closed without merging**